ap/production/normalization.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+
+"""
+Column production methods related to sample normalization event weights.
+"""
+
+import law
+import order as od
+
+from ap.production import Producer, producer
+from ap.util import maybe_import
+from ap.columnar_util import set_ak_column
+
+ak = maybe_import("awkward")
+
+
+@producer(
+    uses={"LHEWeight.originalXWGTUP"},
+    produces={"normalization_weight"},
+)
+def normalization_weights(
+    events: ak.Array,
+    selection_stats: dict,
```

Comment:
Might be nice to type hint the contents of the selection_stats dict

Reply:
Yup, added in c84111a2.